src/sceneHandler/index.ts

```diff
@@ -168,6 +171,7 @@ export class SceneHandler {
     this.showLoading();
     this.dispose(changeSceneRequestFrom);
     setTimeout(() => {
+      this.titleTextElement.style.zIndex = "-1";
```

Comment:
What is the point of this.titleTextElement.style.zIndex = "-1";
Why is this only called before switching to gameplay and level selection scene?
Why not include this in level-end scene? If not due to gameplay, should we call it twice since you already called it in level selection?

Reply:
@bernhardccodev i remove this code to gameplay since it dont need it. I only need to hide the title when switching to level selection